pkg/apis/scheduling/v1beta1/types.go

```diff
@@ -117,6 +120,9 @@ const (
 
 	// NotEnoughPodsOfTaskReason is probed if there're not enough pods of task compared to `spec.minTaskMember`
 	NotEnoughPodsOfTaskReason string = "NotEnoughPodsOfTask"
+
+	// UnEnqueueableReason is probed if job is rejected to enqueue
+	UnEnqueueableReason string = "Unenqueueable"
```

Comment:
I think the status is `Unenqueueable`, the reason is something like `resource is insufficient`

Reply:
But there are multiple reasons in different plugins with status "Unenqueueable", such as NotEnoughQuota, clusterOverCommit. The event in each plugin will record the detail reason why not enqueueable.

Reply:
And the message will also show the detail reason in each plugin:
- extender: "extender return error + error's detail"
- overcommit: "resource in cluster is overused"
- proportion: "queue resource quota insufficient"
- resourcequota: "resource quota insufficient"

Reply:
As the k8s's rule, `reason` is a brief word to explain, and `message` has the detail description.